lib/magma/commands.rb

```diff
@@ -2,6 +2,42 @@
 require 'logger'
 
 class Magma
+  class LoadProject < Etna::Command
+    def execute(file_name)
+      file = File.open(file_name)
+      file_data = JSON.parse(JSON.parse(file.read))
+
+      db = Magma.instance.db
+
+      models = file_data["models"]
+
+      models.keys.each do |model|
+        model_name = models[model]["template"]["name"]
+        attributes = models[model]["template"]["attributes"]
+
+        attributes.keys.each do |att|
+          attribute = attributes[att]
+
+          db[:attributes].insert(
+            project_name: 'i dont know where to find the project name',
+            model_name: model_name,
+            attribute_name: attribute["name"],
+            type: attribute["type"] || "project",
+            display_name: attribute["display_name"],
+            # attribute["shown"] is the opposite of hidden?
+            hidden: attribute["shown"],
+            validation: attribute["validation_object"],
```

Comment:
If you rename `validation_object` to `validation`, alias `name` to `attribute_name`, and `shown` to `hidden`, you could just paste in the attribute directly here.